### PR TITLE
WD-11971 - add new tab to /download/risc-v

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

DO NOT MERGE UNTIL Jul 9th
Added new tab to /download/risc-v

## QA

- [demo link](https://ubuntu-com-14016.demos.haus/download/risc-v)
- [copy doc](https://docs.google.com/document/d/1kkJeq9HdH3iWV35HwihcJd3CxZw9Jy21W66lJl8VbWs/edit)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the section is correct according to copy doc

## Issue / Card
[WD-11971](https://warthogs.atlassian.net/browse/WD-11971)

[WD-11971]: https://warthogs.atlassian.net/browse/WD-11971?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/download/risc-v/microchip-curiosity.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="col-6 risc-v-container u-vertically-center u-align--center">
+      {{ image (
+      url="https://assets.ubuntu.com/v1/e0c220e9-0da5bc43-image001_reworked_20240701.png",
+      alt="Microchip Curiosity",
+      width="452",
+      height="300",
+      hi_def=True,
+      loading="lazy",
+      attrs={"class": "risc-v-image"}) | safe
+      }}
```
Comment (britneywwc) [inline templates/download/risc-v/microchip-curiosity.html:19]:
```suggestion
      {{ image(url="https://assets.ubuntu.com/v1/e0c220e9-0da5bc43-image001_reworked_20240701.png",
                      alt="Microchip Curiosity",
                      width="452",
                      height="300",
                      hi_def=True,
                      loading="lazy",
                      attrs={"class": "risc-v-image"}) | safe
      }}
```

File: templates/download/risc-v/microchip-curiosity.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <h3 class="p-heading--5 u-no-padding--top">
+        Ubuntu Server
+        <br class="u-hide--small" />
+        preinstalled image
+      </h3>
+    </div>
+    <div class="col-6">
+      <p>
+        <a class="p-button--positive"
+           href="https://people.canonical.com/~platform/images/microchip/pic64gx1000_curiosity_kit/ubuntu-24.04-preinstalled-server-riscv64+pic64gx.xz">Download 24.04</a>
```
Comment (britneywwc) [inline templates/download/risc-v/microchip-curiosity.html:34]: The download link is giving a 404. Perhaps the copy doc link is outdated, I'll leave a message on the docs to see if there's a new link for this.

**Review by britneywwc — COMMENTED** [review]
LGTM, just a small suggestion :)

**Review by britneywwc — APPROVED** [review]

File: templates/download/risc-v/microchip-curiosity.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </p>
+      <p>
+        <a href="&nbsp;&rsaquo;">How to install Ubuntu on the Microchip PIC64GX1000 Curiosity Kit</a>
```
Comment (akbarkz) [inline templates/download/risc-v/microchip-curiosity.html:36]: @lizzochek What kind of link is this?

**Comment by lyubomir-popov** [comment]
can we please use the p-section--hero for this? too much space in the hero as it is now:
image

**Pull request merged**